proj/src/imagedata.cpp

```diff
@@ -7,22 +7,26 @@
 
 void imagedata::img2inputdata()
 {
-   
-    m_data.reserve(m_height * m_width);
-
+    //m_data.reserve(m_height * m_width);
+    m_data = std::vector<std::vector<double>>(m_height * m_width, std::vector<double>(5, 0)); 
     for (std::size_t i=0; i < m_height; i++)
     {
         for (std::size_t j=0; j < m_width; j++)
         {
             //std::size_t index = i * width + j;
-            std::vector<double> value(5);
-            value[0] = double(m_img.at<cv::Vec3b>(i, j)[0]) / 255.0;
-            value[1] = double(m_img.at<cv::Vec3b>(i, j)[1]) / 255.0;
-            value[2] = double(m_img.at<cv::Vec3b>(i, j)[2]) / 255.0;
-            value[3] = double(i) / m_height;
-            value[4] = double(j) / m_width;
-
-            m_data.push_back(value);
+            // std::vector<double> value(5);

```

Comment:
Since you have version control, we generally simply remove code when we don't want them.  It can be recovered from git history.

But you are probably in a process of development and the dead code is not fully dead.  In this case, we usually choose to add a remark describing why having dead code here.